tests/mock_network/churn.rs

```diff
@@ -544,8 +557,8 @@ fn messages_during_churn() {
     let min_section_size = 4;
     let network = Network::new(min_section_size, None);
     let mut rng = network.new_rng();
-    let prefixes = vec![2, 2, 2, 3, 3];
-    let prefixes_len = prefixes.len();
+    let prefixes = vec![2, 2, 2, 2];
```

Comment:
The thing is that this is the starting point.
My thinking was that this would mean we would reach the point where that split occurs to get 2 sections with 3 bit prefix. while the other are not yet and this would no longer be happening since we never merge those.  

But I guess with the existing starting point do have benefit too. so I'll revert that.